recipe/meta.yaml

```
@@ -57,18 +56,23 @@ requirements:
     - click >=7.0,<9
     - cloudpickle <4
     - databricks-sdk >=0.20.0,<1
+    - fastapi <1
     - gitpython >=3.1.9,<4
     - importlib-metadata <9,>=3.7.0,!=4.7.0
     - opentelemetry-api >=1.9.0,<3
     - opentelemetry-sdk >=1.9.0,<3
     - packaging <25
```

```suggestion
    - packaging <26
```
https://github.com/mlflow/mlflow/blob/v3.3.2/libs/skinny/pyproject.toml#L37